### PR TITLE
add device hwdb for IdeaPad Miix510-12ISK

### PR DESCRIPTION
**Pull request by BruceZhang1993**
Add hwdb ACCEL_MOUNT_MATRIX for IdeaPad Miix510-12ISK

### CODE REVIEW

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -328,6 +328,10 @@ sensor:modalias:acpi:SMO8840*:dmi:*:svnLENOVO:pn80XF:pvrLenovoMIIX320*
 # IdeaPad Miix 510
 sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80XE:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+ 
+# IdeaPad Miix 510-12ISK 80U1
+sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80U1:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
```
Comment (poettering) [inline hwdb/60-sensor.hwdb:332]: please merge with the previous entry. it's the same product apparently with the same matrix, even though the internal name might have changed? please merge hence and comment why

**Comment by poettering** [comment]
also, tests don't pass, hwdb file is not properly formatted.

File: hwdb/60-sensor.hwdb (hunk 1 above)
Comment (BruceZhang1993) [reply hwdb/60-sensor.hwdb:332]: Yes, it's almost the same product as the previous entry. But I am not that sure if all pn80* can use the same matrix.

Comment (poettering) [reply hwdb/60-sensor.hwdb:332]: that's not what i am suggesting. But see earlier entries: there can be be multiple match  expressions for the same ACCEL_MOUNT_MATRIX line, and please do that kind of merging. Thanks

**Comment by poettering** [comment]
looks ok, but please extend the comment there, do mention your device model

**Comment by hadess** [comment]
Could you please squash the changes squash on your branch? `git rebase -i origin/master` and use the `fixup` or `squash` option to merge all 3 changes into one, and use `git push -f` to push your changes to your branch.

**Comment by keszybz** [comment]
@hadess it's not really necessary to ask contributors to squash. Github can do that for us when merging, which works just as well as long as we are fine with having one patch in the end. For simple PRs like this this is good enough. We use 'please-squash-on-merge' label to mark such cases.

**Pull request merged**